CDCgov/prime-simplereport
# 3905 feature flag setup

# BACKEND PULL REQUEST

## Related Issue

Implements feature flags that can be controlled from the DB table feature_flag ( #3905 )

## Changes Proposed
Diagram below showcases the setup and the elements that got created as part of this PR.

![SR Feature Flags (1)](https://user-images.githubusercontent.com/103958711/181312737-b2a57cb4-dd68-4d91-a771-ff587c2f4c20.jpeg)


After this PR gets approved and merged to main I will include this diagram in the wiki.

## Testing

- Verify front end calls REST endpoint /feature-flags
- Verify endpoint returns multiplexEnabled flag set to true (and after changing the value in the table the response reflects those values after the 1 min refresh)
- Verify the frontend displays or hides multiplex features accordingly.

## Checklist for Primary Reviewer

- [X] GraphQL schema changes are backward compatible with older version of the front-end
